MinnPost/object-sync-for-salesforce
# Fix PHP 8.0 compatibility

## What does this Pull Request do?

This makes sure the plugin runs and runs successfully in PHP 8.0. This fixes #461.

## How do I test this Pull Request?

- install PHP 8.0
- make sure the plugin works as normal
